src/controller.rs

```diff
@@ -167,6 +167,19 @@ impl MetricsController {
     }
 }
 
+// create a record floating point event
+// event value is float
```

Comment:
Because this function is public, we should document it with 'rustdoc' comments. The best way to do that is to copy lines 141 through 152 and put them at the top of this function. You would, of course, change "an Integer" to "a floating point"

You can read more about rustdoc documentation here: [1]. 

[1] https://doc.rust-lang.org/book/documentation.html